guides/source/active_storage_overview.md

```diff
@@ -1370,7 +1370,10 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   # ...
   def after_teardown
     super
-    FileUtils.rm_rf(ActiveStorage::Blob.service.root)
+    directories = ActiveStorage::Blob.services.names.map do |service_name|
+      ActiveStorage::Blob.services.fetch(service_name).root
+    end
```

Comment:
To me this example shows a need for ` ActiveStorage::Blob.services` to implement `Enumerable` rather than expose `names` so the same can be done as:

```ruby
ActiveStorage::Blob.services.map(&:root)
```
or perhaps 
```ruby
ActiveStorage::Blob.services.some_new_public_reader.map(&:root)
```

but to be fair I haven't worked that closely with Active Storage so may be missing context

Reply:
It looks like `fetch` lazy initializes the storage service class via `Service::Configurator` and this could require additional gems (eg aws-sdk-s3) with their own dependencies. 

`some_new_public_reader` with a different name could iterate over the names and fetch for the user, but a way to get the service names alone to avoid loading code also seems useful?